workers/unity/Assets/Fps/Scripts/UI/Tables/Table.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Fps
```

Comment:
I think `Fps.UI` is a namespace we use now. I suppose we ought to update all these scripts to be in that namespace?

Reply:
yeah, probably need to do a sanity check later on to make sure everything uses the right namespace